web/src/app/browser/tsconfig.json

```diff
@@ -2,13 +2,9 @@
   "extends": "../../tsconfig.dom.json",
 
   "compilerOptions": {
-    "baseUrl": "./",
     "outDir": "../../../build/app/browser/obj/",
     "rootDir": "./src",
     "tsBuildInfoFile": "../../../build/app/browser/obj/tsconfig.tsbuildinfo",
-    "paths": {
-      "keyman/engine/*": [ "../../engine/src/*/index.js" ],
-    }
```

Comment:
Consolidated into web/tsconfig.base.json. Note that child `paths` definitions replace parent ones; they don't extend.